### PR TITLE
SLS-496 Automatically create missing oligarch and ingest tables

### PR DESCRIPTION
**Pull request by pmacko86**
A follower, or a leader that restarted without local files, will receive `oligarch:` metadata and automatically create the missing ingest table.

### CODE REVIEW

File: test/suite/test_oligarch15.py [hunk 1]
```diff
@@ -56,6 +56,21 @@ def conn_extensions(self, extlist):
     def early_setup(self):
         os.mkdir('kv_home')
 
+    # Restart the node without local files
+    def restart_without_local_files(self):
+        # Close the current connection
+        self.close_conn()
+
+        # Remove local files
+        for f in os.listdir():
```
Comment (quchenhao) [inline test/suite/test_oligarch15.py:65]: Can we rename instead of delete to ease debugging?

**Review by quchenhao — APPROVED** [review]
lgtm

**Review by korteland — APPROVED** [review]
Nice work! LGTM.

File: test/suite/test_oligarch15.py (hunk 1 above)
Comment (pmacko86) [reply test/suite/test_oligarch15.py:70]: Good idea! Done in [3dfd18f](https://github.com/wiredtiger/wiredtiger/pull/11250/commits/3dfd18f463cde9f0a44a06a80622d307400eb262).

**Pull request merged**